about.html

```diff
@@ -0,0 +1,171 @@
+        <h2>ABREHOT Library</h2>
+        <div class="border-1 about-intro bg-gray">
+          <p>
+            The Abrehot Library, established by Prime Minister Abiy Ahmed, is a
+            modern facility designed to provide a range of resources and
+            services to the public. The library offers a wide range of books
+            that cover various topics, from politics and history to literature
+            and science, available for borrowing. Additionally, it has a unique
+            feature, which is the reading corner. This cozy and inviting space
+            is designed to encourage reading and reflection, offering
+            comfortable chairs and cushions, and natural light, creating a warm
+            and welcoming atmosphere.
+          </p>
+          <p class="none-mobile">
+            Another noteworthy feature of the Abrehot Library is the enjoyable
+            coffee shop. It is a social hub, offering a range of high-quality
+            beverages and snacks, designed for people to relax and socialize.
+            The coffee shop's aim is to create an enjoyable and comfortable
+            space for people to build relationships, share experiences, and
+            discuss ideas. The coffee shop makes the Abrehot Library not only a
+            place for individual learning but also a community resource that
+            fosters social cohesion and civic engagement. In addition, the
+            library provides aud
+          </p>
+        </div>
+        <article id="contact">
+          <p class="bg-gray">
+            Please contact us per Email for any further questions about Abrehot
+            Library <br />
+            <a href="#">yonasle@abrehot.com.et</a>
+          </p>
+        </article>
+      </section>
+      <section class="about-main">
+        <h2>Abrehot Library Corners</h2>
+        <div class="dash">___</div>
+        <p>
+          A comfortable reading corner is a cozy and inviting space designed for
+          relaxation and enjoyment of reading. It typically includes a
+          comfortable chair or sofa, a soft throw or blanket, a good reading
+          light, and a small table or shelf to hold books and other reading
+          materials. The atmosphere is often enhanced by the addition of plants,
+          artwork, and other decorative elements that create a soothing and
+          peaceful ambiance. The reading corner can be located in any part of
+          the house that offers a quiet and secluded space, such as a bedroom,
+          living room, or even a dedicated home library. Whether for leisurely
+          reading, studying, or simply taking a break from the daily hustle and
+          bustle, a comfortable reading corner can provide a perfect haven for
+          relaxation and mental rejuvenation.
+        </p>
+      </section>
+      <!--About page Image Section -->
+      <section class="image-container">
+        <h2>Our Gallery of Abrehot</h2>
```

Comment:
- [ ] Kindly as you did it for the `about-main` by adding the red divider small line, I would suggest to same here to stick to the design template.

### Your design :point_down: 
![Screenshot from 2023-04-27 18-57-41](https://user-images.githubusercontent.com/89228743/234965223-3c4cbc4b-2b93-4a62-8c10-7f570aa71ced.png)

### The template :point_down: 
![Screenshot from 2023-04-27 18-57-24](https://user-images.githubusercontent.com/89228743/234965207-42a4cddf-627d-452b-a440-ebe4cd1505ba.png)

```suggestion
        <h2>Our Gallery of Abrehot</h2>
        <div class="dash">___</div>
```